File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/images/UpdateBmpPropertiesWithoutStorage.java
package com.aspose.imaging.cloud.examples.images;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdateBmpPropertiesWithoutStorage {

    public static void main(String... args) throws IOException {
        String input = "Sample1.bmp";
        String output = "Sample2.bmp";
        Path inputFile = Utils.getPath(UpdateBmpPropertiesWithoutStorage.class, input);
        Path outputFile = Utils.getPath(UpdateBmpPropertiesWithoutStorage.class, output);

        Integer bitsPerPixel = 24;
        Integer horizontalResolution = 300;
        Integer verticalResolution = 300;
        Boolean fromScratch = false;
        String outPath = "";

        com.aspose.imaging.model.ResponseMessage sr = Utils.getImagingSdk().PostImageBmp(
                bitsPerPixel,
                horizontalResolution,
                verticalResolution,
                fromScratch,
                outPath,
                inputFile.toFile()
        );

        Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/DotNET/SDK/VisualBasic/Images/UpdateBmpPropertiesWithoutStorage.vb
'////////////////////////////////////////////////////////////////////////
' Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
'
' This file is part of Aspose.Imaging. The source code in this file
' is only intended as a supplement to the documentation, and is provided
' "as is", without warranty of any kind, either expressed or implied.
'////////////////////////////////////////////////////////////////////////

Imports System
Imports Aspose.Cloud

Namespace Aspose.Imaging.Cloud.Examples.Images
	Friend Class UpdateBmpPropertiesWithoutStorage
		Shared Sub Main()
			Dim dataDir As String = Aspose.Imaging.Cloud.Examples.Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
			Dim input As String = "sample1.bmp"
			Dim output As String = "output.bmp"

			Dim bitsPerPixel As Integer = 24
			Dim horizontalResolution As Integer = 300
			Dim verticalResolution As Integer = 300

			Common.GetImagingSdk().Bmp.UpdateParametersOfBmpImage(bitsPerPixel, horizontalResolution, verticalResolution, dataDir & output, dataDir &input)
		End Sub
	End Class
End Namespace
